website/api/controllers/entrance/signup.js

```diff
@@ -114,7 +120,24 @@ the account verification message.)`,
         stripeCustomerId
       });
     }
-
+    // Send a POST request to Zapier
+    await sails.helpers.http.post(
+      'https://hooks.zapier.com/hooks/catch/3627242/bqsf4rj/',
+      {
+        'emailAddress': newEmailAddress,
+        'organization': signupReason === 'Buy a license' ? organization : '?',
+        'firstName': signupReason === 'Buy a license' ? firstName : '?',
+        'lastName': signupReason === 'Buy a license' ? lastName : '?',
+        'signupReason': signupReason,
+        'webhook-secret': sails.config.custom.zapierSandboxWebhookSecret
```

Comment:
Any reason this shouldn’t be camelCased to match the others?

Reply:
I can't think of any, I updated the name.